### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by pmaytak**
Microsoft Open Source recommends to have Security.md file. Copied the text from their template. Adds a little more details than 'Security Reporting' section in ReadMe.

### CODE REVIEW

**Review by jmprieur — APPROVED** [review]
LGTM.
Thanks @pmaytak

**Comment by keegan-caruso** [comment]
Thanks @pmaytak 

@GeoK  or @brentschmaltz do we want to replace the security reporting section in the main readme with a link to this document?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://www.microsoft.com/msrc).
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
+
+  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
+  * Any special configuration required to reproduce the issue
+  * Step-by-step instructions to reproduce the issue
```
Comment (brentschmaltz) [inline SECURITY.md:22]: I would suggest 'Step-by-step ...' as the first bullet, the 'Proof-of-concept ...'

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.3 BLOCK -->
```
Comment (brentschmaltz) [inline SECURITY.md:1]: @keegan-caruso yes on updating our readme.md. 
We may want to consider replacing the security reporting section to this new page. 

I believe the section "Security Vulnerability in ..." should be moved to a less prominent location as it is almost 3 years old. Perhaps we need a separate page for these items.

**Review by GeoK — APPROVED** [review]

**Pull request merged**